nixos/modules/services/x11/desktop-managers/kodi.nix

```diff
@@ -20,7 +20,7 @@ in
     services.xserver.desktopManager.session = [{
       name = "kodi";
       start = ''
-        ${pkgs.kodi}/bin/kodi --lircdev /run/lirc/lircd --standalone &
+        LIRC_SOCKET_PATH=/run/lirc/lircd ${pkgs.kodi}/bin/kodi --standalone &
```

Comment:
Did that change with an upgrade of Kodi?

Reply:
Lirc support is broken in nixos since more than two releases.

The argument --lircdev was removed with following commit, but broke lirc support:
https://github.com/xbmc/xbmc/commit/6e156bea5bf3a7fb8967e888238b6a07b967a90b

and following commit fixed it again:
https://github.com/xbmc/xbmc/commit/4ad8df65cd6e0987bbca84f0f1e477931532f186

These commits are both 2 years old.